Plan/common/src/main/resources/assets/plan/web/css/style.css

```diff
@@ -1195,4 +1195,21 @@ div.scrollbar {
 .white {
     color: #FFFFFF;
     text-shadow: 0 0 8px #000;
+}
+
+/*
+Adding a translation, even though it doesn't visually
+impact the page, causes child elements with a
+position of "fixed" to be based on this element
+instead of any other parent elements. For example,
+if a child element to this element had `top: 0`
+and `left: 0`, then it would align with the top
+left of this element instead of another parent or the document body.
+See this StackOverflow answer: https://stackoverflow.com/a/20830413
+
+We use this to properly align the popover when there are more
+than 3 events in one day on the sessions calendar.
+ */
+#sessions-overview {
+    transform: translateX(0);
```

Comment:
The popover problem is a tricky one.

I don't unfortunately know the answer, but with quick search through the fullcalendar docs found an alternative if the issue can not be solved - https://fullcalendar.io/docs/moreLinkClick The more click could instead go to Week view which shows all the events properly. If you choose this route the month and also a week button might be needed for the server calendar.

Reply:
I tried applying it to the calendar itself, but then the script just uses the coordinates of the window inside of the calendar div, so the popover would always be down and to the right of where it is supposed to be. I was thinking of just applying some sort of right margin in the same breakpoint where the sidebar will be active, but I am not sure if that would move a fixed position element (haven't tested it yet).

Reply:
Figured it out! On desktop screen sizes (using a media query) I can apply a negative margin to the left side of the popover. This moves it to the left by whatever amount I choose, in this case 14rem or the width of the sidebar. No JS required!